.gitignore

```diff
@@ -23,3 +23,5 @@ docs/doxygen_sqlite3.db
 docs/html/
 # sphinx & breathe output
 docs/build/
+#clion files
+.idea/
```

Comment:
I think that would be better (if this needs to be added at all)

Reply:
I overlooked that is even here. It would be definitely practical to have it, this are just project files in  jetbrains IDEs. Though I could just add it locally to  `.git/info/exclude`.  Yes, a separate PR makes much more sense.